runtime/moonbase/tests/xcm_mock/relay_chain.rs

```diff
@@ -372,7 +374,9 @@ pub(crate) fn relay_events() -> Vec<RuntimeEvent> {
 		.collect::<Vec<_>>()
 }
 
+use crate::xcm_mock::statemint_like::PolkadotXcm;
```

Comment:
Given that we are configuring the relay mock here, shouldn't we import this from `xcm_mock::relay_chain::XcmPallet`?